examples/src/main/python/ml/prefixspan_example.py

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
ditto for java and scala and python example

Reply:
@felixcheung I don't think the doc, java/scala/python example are in 2.4. Shall I open a separate jira to merge in the fix?

Reply:
yes, ping me on that and I can merge quickly